script/Ciel/Day 5/05_01_CIEL05_3C.txt

```diff
@@ -542,12 +586,13 @@ I can't kill Vlov alone.
 [sha:82bb352f096bfd98496a4335790e954567525f60]{
 -- Page 534, Offset 12248.
 -- どんな手を使おうと、あのホテルの惨劇を繰り返す訳にはいかない。だから―――
-No matter what, I can't let the tragedy of what happened at the hotel repeat again. That's why―――
+And no matter what it takes, I can't allow a repeat of the tragedy that happened at the hotel.
+ That's why―――
 }
 [sha:054c650ad6be9cdd0ef2598f9524d7e5edfb28e8]{
 -- Page 535, Offset 12249.
 -- 「―――分かった。頼むアルクェイド。
-"―――I understand. Arcueid―――
+"I understand. Arcueid, I'm asking you―――
```

Comment:
I'd prefer "Arcueid, this is my request."

Don't feel strongly about it though.